charts/grafana/templates/image-renderer-deployment.yaml

```diff
@@ -58,7 +58,7 @@ spec:
       {{- with .Values.imageRenderer.priorityClassName }}
       priorityClassName: {{ . }}
       {{- end }}
-      {{- with .Values.imageRenderer.image.pullSecrets }}
+      {{- with (or .Values.global.imagePullSecrets .Values.imageRenderer.image.pullSecrets) }}
       imagePullSecrets:
         {{- range . }}
         - name: {{ tpl . $root }}
```

Comment:
Would it be possible to re-use the logic here?

https://github.com/RaphSku/grafana-helm-charts/blob/c2455a342053970587bdb32f09ef24bc379d5d67/charts/grafana/templates/_pod.tpl#L314-L317

Reply:
Good idea, have re-written that part.